.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules/
 
 # Config
 config.js
+
+package-lock.json
```

Comment:
I'm not super clear on the specifics, but this stackoverflow article recommends against adding the packag-lock.json to gitignore. 
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

Reply:
It actually didn't work. I didn't realize that this configuration disabled the linter until I started implementing a new component and saw that  I wasn't getting any linter error.
I changed the config again, and now it's working. 
Can you review it for me again, please?

Reply:
Hey @aaronfinsrud, just know I realized I answered the question about the linter where you were asking about the package-lock.json. 
I didn't know it was recommended to commit the package-lock.json as it changes every time you run 'npm install' and causes merging conflicts. I'll take it out of .gitignore as recommended.